tests/java/CheckstyleBearTest.py

```diff
@@ -31,3 +32,13 @@ def test_with_custom_configfile(self):
         self.section["checkstyle_configs"] = self.empty_config
         self.check_validity(self.uut, [], self.good_file)
         self.check_validity(self.uut, [], self.bad_file)
+
+
+def test_known_checkstyle(monkeypatch):
```

Comment:
What is monkeypatch ?
Possibly use unittest.mock.patch ? Then it can be inside the TestCase class and that's what we do in other places and also you wouldn't need to change `from bears.java.CheckstyleBear import CheckstyleBear`